src/Apps/Collect/Components/Seo/SeoProductsForArtworks.tsx

```diff
@@ -14,7 +14,8 @@ const AVAILABILITY = {
   sold: "https://schema.org/OutOfStock",
 }
 
-const formatCurrency = value => currency(value, { separator: "" }).format()
+const formatCurrency = value =>
+  currency(value / 100, { separator: "" }).format()
```

Comment:
Is this safe? Specifically for the currencies that aren't divisible by 100. cc @yuki24 cause I know he's currently in the currency working group.

Reply:
`currency` handles this

Reply:
We should probably be more careful about this (or JPY should be normalized to align with other currencies). I'm looking up works with other currencies such as KRW and CNY and will report back.

![currencies](https://user-images.githubusercontent.com/386234/62244249-08364800-b3ad-11e9-8205-eac6dee9444b.png)

Reply:
Can we merge this PR in the meantime?

Reply:
South Korean won is also not divided:

![Screen Shot 2019-07-31 at 4 25 10 PM](https://user-images.githubusercontent.com/386234/62245533-d2469300-b3af-11e9-891b-fbeb00e02cac.png)

Reply:
Please go ahead and merge.